src/Altinn.App.Core/Internal/Pdf/PdfOptionsMapping.cs

```diff
@@ -0,0 +1,178 @@
+using Altinn.App.Core.Features.Options;
+using Altinn.App.Core.Models;
+using Newtonsoft.Json.Linq;
+
+namespace Altinn.App.Core.Internal.Pdf;
+
+/// <summary>
+/// Default implementation of IPdfOptionsMapping
+/// </summary>
+public class PdfOptionsMapping: IPdfOptionsMapping
```

Comment:
Just a thought: Moving forward we could consider creating a set of domain classes representing the various config files we have as this kind of reading is "business logic".